client/python/gradio_client/version.txt

```diff
@@ -1 +1 @@
-0.2.10
```

Comment:
We should increase the version of the `gradio_client` dependency in `requirements.txt`